File: DSA/MuskanVishwakarma/Day1/Q1_Leetcode.java
class Solution {
    int array_sum;
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int sum = n*(n+1)/2;
        for(int i = 0;i<n;i++){
            array_sum += nums[i];
        }
        int missing = sum-array_sum;
        return missing;
    }
}

File: DSA/Prarthana_Baisoya_590024838/Day1/q1_leetcode.c
#include <stdio.h>

int missingNum(int nums[], int numSize) {
    int expSum = numSize * (numSize + 1) / 2;
    int actualSum = 0;

    for(int i = 0; i < numSize; i++) {
        actualSum += nums[i];
    }

    return expSum - actualSum;
}

int main() {
    int nums[] = {3, 0, 1};
    int size = sizeof(nums) / sizeof(nums[0]);

    printf("Missing Number: %d\n", missingNumber(nums, size));

    return 0;
}
